upstream-community-operators/postgres-operator/postgres-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,277 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: postgres-operator.v1.1.0
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Database
+    certified: "false"
+    repository: https://github.com/zalando/postgres-operator
+    containerImage: registry.opensource.zalan.do/acid/smoke-tested-postgres-operator:v1.0.0-37-g2422d72
+    createdAt: 2019-01-29T11:10:00Z
+    description: Postgres operator creates and manages PostgreSQL clusters running in Kubernetes.
+    support: Zalando SE
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "acid.zalan.do/v1",
+          "kind": "postgresql",
+          "metadata": {
+            "name": "acid-minimal-cluster"
+          },
+          "spec": {
+            "databases": {
+              "foo": "zalando"
+            },
+            "numberOfInstances": 2,
+            "postgresql": {
+              "version": "10"
+            },
+            "teamId": "ACID",
+            "users": {
+              "foo_user": [],
+              "zalando": [
+                "superuser",
+                "createdb"
+              ]
+            },
+            "volume": {
+              "size": "1Gi"
+            }
+          }
+        }
+      ]
+spec:
+  version: 1.1.0
+  minKubeVersion: 1.10.0
+  maturity: stable
+  displayName: Postgres-Operator
+  description: |
+    The Postgres operator manages PostgreSQL clusters on Kubernetes:
+
+    1.The operator watches additions, updates, and deletions of PostgreSQL cluster manifests and changes the running clusters accordingly. For example, when a user submits a new manifest, the operator spawns a new Postgres cluster with necessary entities such as StatefulSets, Services, and also Postgres roles. See this Postgres cluster manifest for settings that a manifest may contain.
+
+    2. The operator also watches updates to its own configuration and alters running Postgres clusters if necessary. For instance, if a pod Docker image is changed, the operator carries out the rolling update. That is, the operator re-spawns one-by-one pods of each StatefulSet it manages with the new Docker image.
+
+    3. Finally, the operator periodically synchronizes the actual state of each Postgres cluster with the desired state defined in the cluster's manifest.
+
+    4. The operator aims to be hands free and configuration happens only via manifests and its own config. This enables easy integration in automated deploy pipelines with no access to Kubernetes directly.
+    ```
```

Comment:
Seems like extra backticks and quotes were left at the end of the description. These can be removed, unless it's some form of markdown I'm not aware of

Reply:
Yup. Removed.